### ISSUE TITLE
Listing Creation - Approve Error

### ISSUE THREAD

**Issue by sarahlazarich** [metrotranscom/doorway#639]
**Bug:**
Existing bug where clicking approve and publish form the details view doesn't surface an error even if there is one. It will appear to do nothing	

**Expected Behavior:** 
Should show an error and encourage user to update listing and we should consider improved validation for a partner to submit a listing

**Comment by sarahlazarich** [comment]
We should show some messaging on the details part 
This is a part of our whole error validation epic / effort 
This is detail --> approve & publish, not edit --> approve and publish
Might need a little design support to see if this is a message or a toast (Can we keep it consistent with what's on the edit page) and Em to make the final call.

**Comment by sarahlazarich** [comment]
Wait on design choice to point

**Comment by sarahlazarich** [comment]
@sarahlazarich & Em to add copy 
This will show on the detail view, not route to the edit view

**Comment by sarahlazarich** [comment]
Copy here should be: 

There are errors in this listing that must be resolved before publishing. To see the errors, please try to approve this listing from the edit view.

**Comment by ludtkemorgan** [comment]
QA Notes:
- As a partner user submit a listing that is missing at least one required field.
- Log in as an admin and click the "Approve & Publish" button for that listing (note: not on the edit view, but on the page that shows when first going to a listing in partners)
- New error message should show

**Comment by sarahlazarich** [comment]
@ludtkemorgan what env should I QA this in?

tried [here](https://partners.housingbayarea.bloom.exygy.dev/listings/7f1ad61e-9b60-45a3-904d-2e7181c39545) & Doorway and didn't see the new error in either spot.

**Comment by ludtkemorgan** [comment]
@sarahlazarich It's only in core as of now. so https://partners.bloom.exygy.dev/. I'm blocked from merging it in Doorway until another PR is merged since it touches the same spot

**Comment by sarahlazarich** [comment]
@ludtkemorgan let's talk through this one this am. I might be missing something, but as a Partner user in https://partners.bloom.exygy.dev/, I'm able to publish directly without submitting my listing for approval.

**Comment by ludtkemorgan** [comment]
@sarahlazarich sounds good. Looks like we don't have the listing approval feature turned on for that environment

**Comment by sarahlazarich** [comment]
ok, lgtm
